prometheus/README.md

```diff
@@ -60,6 +60,35 @@ The provided `values.yaml` covers the following adjustments:
 - Client certificate injection to support scraping of workload secured with Istio strict mTLS
 - Active scraping of workload annotated with prometheus.io/scrape
 
+    >**Note:** If the adminssion webhooks are also being installed then these webhooks should exclude resources being installed in the `kyma-system` namespace. This can be done in the following way:
```

Comment:
```suggestion
    >**Note:** If the admission webhooks are also being installed, then these webhooks must exclude resources being installed in the `kyma-system` Namespace. This can be done in the following way:
```